### PR TITLE
Pass symbolicator config as YAML

### PR DESCRIPTION
**Pull request by philicious**
This adds more config options to symbolicator deployment.

We needed `connect_to_reserved_ips` , which wasnt supported, so I took the chance and added more along the way.

I tested symbolicator to start properly also with the `caches` enabled

### CODE REVIEW

File: sentry/templates/configmap-symbolicator.yaml [hunk 1]
```diff
@@ -14,8 +14,21 @@ data:
     cache_dir: "/data"
     bind: "0.0.0.0:{{ template "symbolicator.port" }}"
     logging:
-      level: "{{ .Values.symbolicator.api.logging.level }}"
+      level: {{ .Values.symbolicator.api.logging.level | quote }}
     metrics:
-      statsd: null
-    sentry_dsn: null # TODO: Automatically fill this with the internal project DSN
+    {{- if .Values.symbolicator.api.statsd.server }}
```
Comment (Mokto) [inline sentry/templates/configmap-symbolicator.yaml]: Maybe we should make this logic super simple by just passing a yaml from the values file ?
With toYaml

It feels we need to add each config in the repository every time a new one is available.

Comment (philicious) [reply sentry/templates/configmap-symbolicator.yaml]: Actually true. Will update the PR in next days

Comment (philicious) [reply sentry/templates/configmap-symbolicator.yaml]: @Mokto I updated the PR. Symbolicator starts fine with the default values config and also works for me when passing custom config.
should be a backwards compatible change as the defaults didnt change

**Comment by philicious** [comment]
@Mokto I could add some smybolicator paragraph to README as it wasnt totally straight-forward to get it working, like
```
        filestore:
          filesystem:
            persistence:
              persistentWorkers: true
```
but also needed to set `connect_to_reserved_ips: true` 

maybe its even worth changing `connect_to_reserved_ips` to default true?

**Comment by Mokto** [comment]
I agree with both.

I'll let you update your PR and then I'll merge it.


Thanks!

**Comment by philicious** [comment]
updated README and also default value of `connect_to_reserved_ips` @Mokto

File: sentry/README.md [hunk 2]
```diff
@@ -77,3 +81,23 @@ For your security, the [`system.secret-key`](https://develop.sentry.dev/config/#
...
+
+## Symbolicator
+
+For getting native stacktraces and minidumps symbolicated with debug symbols (e.g. iOS/Android), you need to enable Symbolicator via
+```yaml
+symbolicator:
+  enabled: true
+```
+
+However, you also need to share the data between sentry-worker and sentry-web via
```
Comment (Mokto) [inline sentry/README.md]: This is not the recommended way. 
Recommended way is to use something else than filestore, like S3 or GCS.

Please adapt your description with this ;)

Comment (philicious) [reply sentry/README.md]: hm yes I agree. however S3 is not supported by the chart. Another option I forgot to mention is using an AWS EFS Storageclass, which supports ReadWriteMany.
I will update the doc and make the options more clear

Comment (Mokto) [reply sentry/README.md]: https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml#L359
https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml#L398-L405

S3 is supported by the chart 👍

Comment (philicious) [reply sentry/README.md]: Hm ye I saw it in values but imho I didn't find a reference in the templates or did I miss it?
(Wouldn't have worked in my personal case tho as we cannot use IAM users/access-keys but need assumable roles and so on)

Comment (philicious) [reply sentry/README.md]: @Mokto I updated again to be more precise about the storage options and caveats

**Comment by Mokto** [comment]
Thaaaanks ! 👍

**Pull request merged**